improver_tests/ensemble_copula_coupling/ensemble_copula_coupling/test_EnsembleReordering.py

```diff
@@ -328,21 +191,17 @@ def test_ordered_data(self):
         """
         raw_data = np.array(
             [
-                [[[1, 1, 1], [1, 1, 1], [1, 1, 1]]],
-                [[[2, 2, 2], [2, 2, 2], [2, 2, 2]]],
-                [[[3, 3, 3], [3, 3, 3], [3, 3, 3]]],
+                [[1, 1, 1], [1, 1, 1], [1, 1, 1]],
+                [[2, 2, 2], [2, 2, 2], [2, 2, 2]],
+                [[3, 3, 3], [3, 3, 3], [3, 3, 3]],
             ]
         )
 
         calibrated_data = raw_data
+        raw_cube = self.cube.copy(data=raw_data)
+        calibrated_cube = self.cube.copy(data=calibrated_data)
```

Comment:
Why are we setting ```calibrated_data = raw_data ``` and then using it in this way? Just use ```raw_data```.

Reply:
OK